gson/src/test/java/com/google/gson/JsonArrayAsListTest.java

```diff
@@ -107,38 +85,22 @@ public void testAdd() {
     assertThat(list.add(JsonNull.INSTANCE)).isTrue();
 
     List<JsonElement> expectedList = Arrays.<JsonElement>asList(
-        new JsonPrimitive(2),
-        new JsonPrimitive(3),
-        new JsonPrimitive(1),
-        new JsonPrimitive(4),
-        JsonNull.INSTANCE
+            new JsonPrimitive(2),
+            new JsonPrimitive(3),
+            new JsonPrimitive(1),
+            new JsonPrimitive(4),
+            JsonNull.INSTANCE
```

Comment:
I don't think this indentation change is correct, could you please revert it?
```suggestion
        new JsonPrimitive(2),
        new JsonPrimitive(3),
        new JsonPrimitive(1),
        new JsonPrimitive(4),
        JsonNull.INSTANCE
```